### PR TITLE
Firebase

### PR DESCRIPTION
**Pull request by curiousily**
* Replace Realm + Server API with Firebase
* Local cache for Firebase is disabled due to bug in the SDK
* Anonymous login (only)

### CODE REVIEW

File: app/src/main/java/io/ipoli/android/app/App.java [hunk 1]
```diff
@@ -268,63 +279,62 @@ private void registerServices() {
...
+        initAppStart();
+    }
+
+    @Subscribe
     public void onScheduleRepeatingQuests(ScheduleRepeatingQuestsEvent e) {
-        scheduleQuestsFor4WeeksAhead().compose(applyAndroidSchedulers()).subscribe(quests -> {
-        }, Throwable::printStackTrace, () ->
-                eventBus.post(new SyncCompleteEvent()));
+        scheduleQuestsFor4WeeksAhead();
+        eventBus.post(new SyncCompleteEvent());
```
Comment (polyvjk) [inline app/src/main/java/io/ipoli/android/app/App.java]: Do we have to delete SyncCompleteEvent?

Comment (curiousily) [inline app/src/main/java/io/ipoli/android/app/App.java]: Yep, we do not handle it anymore.

**Comment by polyvjk** [comment]
Veeeery good job :)

**Comment by curiousily** [comment]
You too 👍

**Pull request merged**